src/coreComponents/mesh/MeshLevel.hpp

```diff
@@ -139,6 +142,17 @@ class MeshLevel : public dataRepository::Group
    */
   ElementRegionManager * getElemManager()             { return &m_elementManager; }
```

Comment:
```suggestion
  ElementRegionManager & getElemManager()             { return m_elementManager; }
```

Reply:
Do you want me to change this in this PR? I can but I ll have to make changes throughout the entire code I guess.